Features/NFT/Sources/Scenes/NFTDetailsScene.swift

```diff
@@ -73,17 +78,57 @@ public struct NFTDetailsScene: View {
         .listSectionSpacing(.compact)
         .navigationTitle(model.title)
         .background(Colors.grayBackground)
+        .modifier(
+            ToastModifier(
+                isPresenting: $showSavedMessage,
+                value: Localized.Nft.successSavedToPhotos,
+                systemImage: SystemImage.gallery
+            )
+        )
+        .alert(Localized.Nft.photoAccessDenied, isPresented: $showPhotoPermissionMessage) {
+            Button(Localized.Common.openSettings) {
+                onSelectOpenSettings()
+            }
+            Button(Localized.Common.cancel, role: .cancel) {}
+        } message: {
+            Text(Localized.Nft.photoAccessDeniedDescription)
```

Comment:
```suggestion
            Text(Localized.Nft.photoAccessDeniedDescription)
```
I would suggest moving to `permission` feature, as we might ask other permissions, or make it generic to avoid too much text